idl-gen/src/type_names.rs

```diff
@@ -20,13 +20,14 @@
 
 use crate::errors::{Error, Result};
 use convert_case::{Case, Casing};
-use sails_rtl::{ActorId, CodeId, MessageId, H160, H256, U256};
+use sails_rtl::{ActorId, CodeId, MessageId, NonZeroU256, H160, H256, U256};
 use scale_info::{
     form::PortableForm, PortableType, Type, TypeDef, TypeDefArray, TypeDefPrimitive,
     TypeDefSequence, TypeDefTuple, TypeInfo,
 };
 use std::{
     collections::{BTreeMap, HashMap},
+    num::{NonZeroU128, NonZeroU16, NonZeroU32, NonZeroU64, NonZeroU8},
```

Comment:
Is it better to use the ones coming from `sails_rtl` as the above `NonZeroU256`?